plugins/soundsourcem4a/soundsourcem4a.cpp

```diff
@@ -562,14 +562,14 @@ ReadableSampleFrames SoundSourceM4A::readSampleFramesClamped(
         } else {
             // Decoded into temporary buffer
             DEBUG_ASSERT(numberOfSamplesDecoded <= decodeBufferCapacity);
-            // Adjust the size of the buffer to the samples that
+            // Shrink the size of the buffer to the samples that
             // have actually been decoded
-            m_sampleBuffer.readFifo(decodeBufferCapacity - numberOfSamplesDecoded);
+            m_sampleBuffer.readFromTail(decodeBufferCapacity - numberOfSamplesDecoded);
```

Comment:
Did you consider to rename this function to shrinkTail or something? The return value is never used.